automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/methodconfigs/WorkspaceMethodConfigDetailsPage.scala

```diff
@@ -37,14 +38,15 @@ class WorkspaceMethodConfigDetailsPage(namespace: String, name: String, methodCo
   def clickLaunchAnalysis(): Unit = {
     openLaunchAnalysisModalButton.doClick()
     // defensive code to prevents test from failing. after click, expect to find either a Message or Analysis Modal
-    implicit val patienceConfig: PatienceConfig = PatienceConfig(timeout = scaled(Span(10, Seconds)), interval = scaled(Span(3, Seconds)))
-    Try[String](eventually[String] {
-      find(CssSelectorQuery("body.broadinstitute-modal-open")).get.underlying.getTagName
-    }) match {
+    Try(
+      await condition (find(CssSelectorQuery(".broadinstitute-modal-open")).nonEmpty, 5)
+    ) match {
       case Failure(e) =>
-        logger.debug(s"Retrying click [button:${openLaunchAnalysisModalButton.query}]")
-        openLaunchAnalysisModalButton.doClick()
+        logger.warn(s"clickLaunchAnalysis failed. Retrying click [button:${openLaunchAnalysisModalButton.query}]")
+        // try alternative click
+        new Actions(webDriver).moveToElement(openLaunchAnalysisModalButton.query.webElement).click().build().perform();
       case Success(some) =>
+        logger.info("clickLaunchAnalysis Success")
```

Comment:
for debugging if fails again at this point. it will be removed in future once fix is confirmed to work.

Reply:
It might be more informative to log if the 'alternative' click above worked.

Reply:
retry click is logged. if retry fails, test would ends unexpectedly quickly. if retry works, test would continue to run. additional log doesn't have too much of value.